sdk/nodejs/cloudwatch/logGroupMixins.ts

```diff
@@ -76,7 +76,7 @@ export class LogGroupEventSubscription extends lambda.EventSubscription {
         name: string, logGroup: logGroup.LogGroup, handler: LogGroupEventHandler,
         args: LogGroupEventSubscriptionArgs = {}, opts: pulumi.ComponentResourceOptions = {}) {
 
-        super("aws:cloudwatch:LogGroupEventSubscription", name, opts);
+        super("aws:cloudwatch:LogGroupEventSubscription", name, { parent: logGroup, ...opts });
```

Comment:
this is our pattern elsewhere.  it allows the caller ot have final say (through ...opts) but allows the current type to say "i'm the appropriate parent otherwise" (by being first in the splat)